twitchio/message.py

```diff
@@ -61,6 +64,15 @@ def __init__(self, **kwargs):
         self._tags = kwargs.get("tags")
         self.echo = kwargs.get("echo", False)
 
+        if self._tags is not None:
+            self.first = self._tags.get("first-msg")
```

Comment:
```suggestion
            first = self._tags.get("first-msg")
```
Assigning it to `self.first` makes it `Optional[bool | str]` if neither of the cases catch (ie if twitch were to use `t`/`f` instead of `1`/`0` for whatever strange reason